crates/solvers/src/tests/naive/without_pool.rs

```diff
@@ -5,7 +5,11 @@ use {crate::tests, serde_json::json};
 
 #[tokio::test]
 async fn test() {
-    let engine = tests::SolverEngine::new("naive", tests::Config::None).await;
+    let engine = tests::SolverEngine::new(
+        "naive",
+        tests::Config::File("config/example.naive.toml".into()),
```

Comment:
Other solver tests don't reuse the example files here but instead pass the config string. That way you can update the example file without breaking any tests.

Reply:
AFAIS this is done in cases where `mock::http::setup` is used

Reply:
> Other solver tests don't reuse the example files here but instead pass the config string

We typically have one test with the example configuration to test it in CI, I agree with @MartinquaXD that for some of the tests at least, we should pass in a configuration string.

Reply:
Ah ok, I substituted all of them with a string instead of one.